cmd/object-api-utils_test.go

```diff
@@ -340,42 +339,34 @@ func TestIsCompressed(t *testing.T) {
 // Tests excludeForCompression.
 func TestExcludeForCompression(t *testing.T) {
 	testCases := []struct {
-		object string
-		header http.Header
-		result bool
+		object      string
```

Comment:
Lets avoid refactoring in this PR, send a different PR for this change.

Reply:
Done. Will send that in a seperate PR after this gets merged.